server/restapi/restapi.coffee

```diff
@@ -0,0 +1,59 @@
+Api = new Restivus
+	useDefaultAuth: true
+	prettyJson: true
+
+
+Api.addCollection ChatMessage
+Api.addCollection ChatRoom
+
+
+Api.addRoute 'version', authRequired: false,
+	get: ->
+		version = {api: '0.1', rocketchat: '0.5'}
+		status: 'success', versions: version
+
+Api.addRoute 'publicRooms', authRequired: true,
+	get: ->
+		rooms = ChatRoom.find({ t: 'c' }, { sort: { msgs:-1 } }).fetch()
+		if rooms
```

Comment:
Added 'real' exception handling for the previous case (pls see commit).
For this case, do we want to 'tighten' the query so that the caller cannot get all the t:'c' info?
I can remove the 'if' otherwise, thx.